### PR TITLE
steamutils001 imp

### PR DESCRIPTION
**Pull request by Detanup01**

### CODE REVIEW

**Comment by universal963** [comment]
Maybe consider having a look at open steamworks? Better than nothing isn't it

**Comment by Detanup01** [comment]
I have checked like 6 of it, one dump has that, others dont have GetIPCountry. Others only have 4

**Comment by otavepto** [comment]
I looked into the returned vftable from real steamclient64.dll and your header is 100% accurate 👍

File: sdk/steam/isteamutils001.h [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+#ifndef ISTEAMUTILS001_H
+#define ISTEAMUTILS001_H
+#ifdef STEAM_WIN32
+#pragma once
+#endif
+
+// Might be not complete/real. Some only has 4, 5, or 7 function, we want them all here.
```
Comment (otavepto) [inline sdk/steam/isteamutils001.h]: this header is accurate/complete 👍

**Pull request merged**